ext/standard/basic_functions.stub.php

```diff
@@ -109,9 +109,9 @@ function min(mixed $arg, mixed ...$args): mixed {}
 
 function max(mixed $arg, mixed ...$args): mixed {}
 
-function array_walk(array|object &$input, callable $funcname, mixed $userdata = UNKNOWN): bool {}
+function array_walk(array|object &$input, callable $callback, mixed $userdata = UNKNOWN): bool {}
```

Comment:
`$user_data` looks better for me

Reply:
Erm.  Did you mean a different entry?  It's a callable, not data.

Reply:
Sorry, I wasn't too clear. The `$callback` name is perfect, but I would also change `$userdata` to `$user_data`, or actually `$arguments` (Nikita probably prefers `$args` :) ), or maybe `$callback_arguments`.

Reply:
Ah, I see what you mean.  $arguments makes the most sense to me, possibly $callback_arguments but that seems needlessly long.  Will update.